src/v2/api/services/ScheduleService.ispec.ts

```diff
@@ -556,9 +589,231 @@ describe('ScheduleService', () => {
                       lesson.disciplineType.name !== DisciplineTypeEnum.LABORATORY
         ))
       ).toBe(true);
-    });  
+    });
   });
-  
+
+  describe('createGroupEvent', () => {
+    it('should create an event without a discipline', async () => {
+      const createEventDTO: CreateEventDTO = {
+        groupId: 'group',
+        name: 'name3',
+        period: Period.NO_PERIOD,
+        startTime: new Date('2022-09-12T08:30:00'),
+        endTime: new Date('2022-09-12T10:00:00'),
+        eventInfo: 'Description of the event',
+        teachers: ['teacherId20'],
+      };
+
+      const result = await scheduleService.createGroupEvent(createEventDTO);
+      expect(result).toStrictEqual({
+        event: {
+          groupId: createEventDTO.groupId,
+          name: createEventDTO.name,
+          period: createEventDTO.period,
+          startTime: createEventDTO.startTime,
+          endTime: createEventDTO.endTime,
+          createdAt: expect.any(Date),
+          updatedAt: expect.any(Date),
+          eventInfo: [
+            {
+              createdAt: expect.any(Date),
+              description: createEventDTO.eventInfo,
+              eventId: result.event.id,
+              number: 0,
+              updatedAt: expect.any(Date),
+            },
+          ],
+          lessons: [],
+          group: {
+            code: 'AA-12',
+            createdAt: expect.any(Date),
+            id: 'group',
+            updatedAt: expect.any(Date),
+          },
+          id: result.event.id,
+          url: result.event.url,
+        },
+      });
+    });
+
+    it('should create an event with a discipline', async () => {
```

Comment:
In this case, the creation of an event with discipline and teachers is tested.
You should check the object of the event and whether teachers have joined this discipline.